src/Core/System/Snippet/Command/InstallTranslationCommand.php

```diff
@@ -40,10 +43,27 @@ protected function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $locales = $this->getLocales($input);
-        $progressBar = $this->createProgressBar($output, \count($locales));
+        $metadata = $this->metadataLoader->getUpdatedMetadata($locales);
+
+        $localesRequiringUpdate = $metadata->getLocalesRequiringUpdate();
+        if ($localesRequiringUpdate === []) {
+            $output->writeln('All translations are up to date.');
```

Comment:
```suggestion
            $output->writeln('All translations are already up to date.');
```
Optional, but like this you can rule out that something happend. Also would be more consistent with the text following.